sql/derived_tables/item_ext.sql

```diff
@@ -38,7 +38,8 @@ WITH items AS (
         json_extract_path_text(data, 'circulationNotes', 'descriptionOfPieces') AS description_of_pieces,
         json_extract_path_text(data, 'status', 'date') AS status_date,
         json_extract_path_text(data, 'status', 'name') AS status_name,
-        json_extract_path_text(data, 'holdingsRecordId') AS holdings_record_id
+        json_extract_path_text(data, 'holdingsRecordId') AS holdings_record_id,
+        json_extract_path_text(DATA, 'discovery_suppress') AS discovery_suppress
```

Comment:
It is probably a good idea to use lowercase for identifiers, i.e. `data` rather than `DATA`.  DBeaver can be configured not to convert them to uppercase, though it is not entirely obvious at first glance how to do it.

Reply:
I fixed that also in my local. I caught the uppercase one in the holdings but not the item. For some reason I can only figure out all caps or no all caps. :(